doc/MONSTERS.md

```diff
@@ -315,6 +316,18 @@ Amount of cutting damage added to die roll on monster melee attack.
 
 Monster protection from bashing, cutting, stabbing, acid and fire damage.
 
+## "weakpoints"
+(array of objects, optional)
+
+Weakpoints in the monster's protection.
+
+| field               | description
+| ---                 | ---
+| `name`              | Name of the weakpoint.
+| `coverage`          | Base probability of hitting the weakpoint. May be increased by skill level.
```

Comment:
Is this number expressed as a decimal probability or a percent? That should probably be clear, like "a 'coverage' of 5 means a base 5% chance of hitting this weakpoint"

Reply:
I changed it to be a percentage, to match other coverage / probability values.